sentry-samples/sentry-samples-spring-boot-jakarta-opentelemetry/README.md

```diff
@@ -0,0 +1,122 @@
+# Sentry Sample Spring Boot 3.0+
+
+Sample application showing how to use Sentry with [Spring boot](http://spring.io/projects/spring-boot) from version `3.0` onwards.
```

Comment:
I think it would be good to add the fact, that this is a sample including OTEL, something like: 
Sample application showing how to use Sentry with [Spring boot](http://spring.io/projects/spring-boot) from version `3.0` onwards and `OpenTelemetry`.

Also, we might want to add some guidance on how to add the otel agent in the readme wdyt?

Reply:
We could also add some instructions on how to run the `system-tests` locally